landing/index.html

```diff
@@ -344,10 +344,12 @@ <h1 class="section__title h1">Виникли запитання?</h1>
         </div>
     </footer>
 
-
     <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r118/three.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.js"></script>
     <script src="https://unpkg.com/ionicons@5.1.2/dist/ionicons.js"></script>
     <script src="static/dist/vendors~main.js"></script>
     <script src="static/dist/main.js"></script>
+    <script src="jquery.js"></script>
+    <script src="jquery.validate.js"></script>
```

Comment:
you dont need this here, i think jquery build exists in dist/main.js or dist/vendors~main.js. Read about **Webpack**.